bigbluebutton-html5/imports/ui/components/user-list/user-list-item/component.jsx

```diff
@@ -288,41 +245,27 @@ class UserListItem extends Component {
       return null;
     }
 
-    let userNameSub = [];
+    const userNameSub = [];
 
-    if (user.isPresenter) {
-      userNameSub.push(intl.formatMessage(messages.presenter));
+    if (user.isLocked) {
+      userNameSub.push(<span>
+        <Icon iconName="lock" />
+        {intl.formatMessage(messages.locked)}
+      </span>);
     }
 
-    if (user.isCurrent) {
-      userNameSub.push(`(${intl.formatMessage(messages.you)})`);
+    if (user.isGuest) {
+      userNameSub.push(intl.formatMessage(messages.guest));
     }
 
-    userNameSub = userNameSub.join(' ');
-
-    // FIX ME
-    /* const { disablePrivateChat,
-            disableCam,
-            disableMic,
-            disablePublicChat } = meeting.roomLockSettings;*/
-
-    const disablePrivateChat = false;
-    const disableCam = false;
-    const disableMic = false;
-    const disablePublicChat = false; // = meeting.roomLockSettings;
-
     return (
       <div className={styles.userName}>
         <span className={styles.userNameMain}>
-          {user.name}
+          {user.name} <i>{(user.isCurrent) ? `(${intl.formatMessage(messages.you)})` : ''}</i>
         </span>
         <span className={styles.userNameSub}>
-          {userNameSub}
-          {(user.isLocked) ?
-            <span> {(user.isCurrent ? ' | ' : null)}
-              <Icon iconName="lock" />
-              {intl.formatMessage(messages.locked)}
-            </span> : null}
+          {userNameSub
+            .reduce((prev, curr) => [prev, ' | ', curr])}
```

Comment:
This line is throwing me errors, whatever i try open the userlist

![screenshot from 2017-07-31 16-57-44](https://user-images.githubusercontent.com/17770979/28795747-675ffcd0-7611-11e7-815a-4c025778d588.png)